### DISCUSSION TITLE
Default Maui App takes 14 seconds to open on iOS.

### DISCUSSION THREAD

**Discussion by aokocax** [dotnet/maui#5172] [Q&A]
Regardless of debug and release, the app takes 14 seconds to open on my iPhone 7 Plus. Android time is much faster but iOS unfortunately is pretty bad. Is there anything that can be done to speed it up?

For Issue
=> https://github.com/dotnet/maui/issues/5183

https://user-images.githubusercontent.com/1059475/157531982-eeb8eacb-3645-4218-823d-43895f7b17b3.MP4

**Comment by Laftek** [comment]
I think it will be faster if you open new issue.

**Comment by aokocax** [comment]
Thanks, I opened it https://github.com/dotnet/maui/issues/5183

**Reply by Laftek** [reply]
Oh sorry didnt see. But I am suprised you got that answer from @jfversluis. I thought its some trivial setting that is causing it (maybe it still is). iPhone 7 Plus is not that old so I really hope its not mirroring real performance of the MAUI.

**Reply by jfversluis** [reply]
I am not saying that is _the_ cause here. I just mentioned there are a lot of factors that can influence this some of them being how the project is configured or that the device is older.

I closed the issue because it's one of those issues that will linger around without any concrete condition to close it. We are continuously monitoring and improving performance across the product. I appreciate you pointing this out and taking the time to report this and even record a video, but without any details it's going to be hard to diagnose.

Sorry if that wasn't clear enough

**Reply by aokocax** [reply]
Thank you Gerald. I sent the application TestFlight and after that when I run it it opened in under 1 second.

**Reply by jfversluis** [reply]
Hm that is weird though... Something with the debugging bits being included then. Glad its not showing when released! Thanks for letting us know!
